packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts

```diff
@@ -525,7 +532,7 @@ export class AreaSeries extends CartesianSeries<
             pointerEvents: PointerEvents.None,
             stroke: this.properties.stroke,
             strokeWidth,
-            strokeOpacity: this.properties.strokeOpacity,
+            strokeOpacity: this.properties.strokeOpacity * (yFilterKey != null ? 0.25 : 1),
```

Comment:
We probably shouldn't hard-code these static values here - perhaps add a hidden `yFilterOpacity` property which defaults to `0.25` instead?